Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.MapViewModel.swift

```diff
@@ -15,7 +15,6 @@
 import ArcGIS
 import Combine
 import Foundation
-import UserNotifications
```

Comment:
Should this be imported in `OfflineMapAreasExampleView.swift`?

Reply:
The notification authorization works without this import. I think the `SwiftUI` import has support for `UserNotifications`, but I'm not sure how to confirm that.

Reply:
I would suggest adding it anyway. It helps make the file's dependencies more explicit, which can aid readability.